crypto/fipsmodule/bcm.c

```diff
@@ -270,8 +270,7 @@ static void BORINGSSL_bcm_power_on_self_test(void) {
 
 #if defined(FIPS_ENTROPY_SOURCE_JITTER_CPU)
   if (jent_entropy_init()) {
-    fprintf(stderr, "CPU Jitter entropy RNG initialization failed.\n");
-    AWS_LC_FIPS_failure("CPU Jitter failed to initilize");
+    AWS_LC_FIPS_failure("CPU Jitter entropy RNG initialization failed");
```

Comment:
> This does not change any of the expected output from AWS-LC.
Doesn't this change the output? Or was this an error from the earlier refactor?

Reply:
Correct to both:
1. The message printing out the error to stderr "CPU Jitter entropy RNG initialization failed" has been there since we added jitter 
2. In https://github.com/aws/aws-lc/commit/5553a20c813aa0b98ff7f26af1822633eccbca20#diff-eb881f93cebfbb804479d55b9d097763dbb1d6e85da9d3d9bfbbf2d9cb1db04d I passed this new slightly different message to AWS_LC_FIPS_failure